lib/build-tokens.js

```diff
@@ -85,49 +110,60 @@ async function buildTokensCommand(commandArgs) {
       : [],
     transform: {
       'color/sass-color-functions': {
-        ...StyleDictionary.transform['color/sass-color-functions'],
-        transformer: (token) => colorTransform(token, themeVariant),
+        ...StyleDictionary.hooks.transforms['color/sass-color-functions'],
+        transform: (token) => colorTransform(token, themeVariant),
       },
     },
-    format: {
-      'css/custom-variables': formatterArgs => createCustomCSSVariables({
-        formatterArgs,
-        themeVariant,
-      }),
-    },
     platforms: {
       css: {
         ...coreConfig.platforms.css,
         files: [
           {
             format: 'css/custom-variables',
             destination: `themes/${themeVariant}/variables.css`,
-            filter: hasSourceTokensOnly ? 'isSource' : undefined,
+            filter: hasSourceTokensOnly
+              ? `isSource.${themeVariant}`
+              : `isThemeVariant.${themeVariant}`,
```

Comment:
This specific `filter` pertains to theme variants, regardless of `hasSourceTokensOnly`.

For example, `isSource.${themeVariant}` is filtering for source tokens of a specific theme variant; similarly, `isThemeVariant.${themeVariant}` is filtering for all tokens (source or otherwise) for a specific theme variant.

`source` tokens are the ones from custom brand packages, i.e. the tokens to extend/modify the default tokens defined within the Paragon repo (passed via style-dictionary's `include`).

When `hasSourceTokensOnly`, it's telling style-dictionary to only transform/output tokens from the `source` tokens (aka only the custom brand package tokens, not the full set of tokens in upstream Paragon).

Reply:
> When `hasSourceTokensOnly`, it's telling style-dictionary to only transform/output tokens from the source tokens (aka only the custom brand package tokens, not the full set of tokens in upstream Paragon).

I guess the word that feels a little odd to me is "has" - maybe `transformSourceTokensOnly` or `outputSourceTokensOnly`?

Reply:
Fair enough. I like the transform prefix suggestion instead 🙂

Reply:
I like `transformSourceTokensOnly` the best so far too! Let's roll with that!

Reply:
https://github.com/openedx/paragon/pull/3186/commits/b4d560bee10f28a9ea2c9d26275b647334a00111